mbrl/util/replay_buffer.py

```diff
@@ -689,3 +689,210 @@ def get_all(self, shuffle: bool = False) -> TransitionBatch:
     @property
     def rng(self) -> np.random.Generator:
         return self._rng
+
+
+class TrajBasedReplayBuffer(ReplayBuffer):
```

Comment:
Is this class being used? From our previous conversation it sounded like you no longer need this. Can we remove?